apps/next/src/content/docs/llamaindex/modules/embeddings/available_embeddings/mistral.mdx

```diff
@@ -2,10 +2,11 @@
 title: MistralAI
 ---
 
```

Comment:
if we need a package, then we also need to tell the user, reuse this widget:
<img width="838" alt="image" src="https://github.com/user-attachments/assets/c0c32e28-eb05-4274-ac1e-b0172416d4ba" />

Reply:
same for the other cases where we need the package